gossip/src/crds_value.rs

```diff
@@ -494,27 +499,77 @@ impl Sanitize for NodeInstance {
 pub struct RestartLastVotedForkSlots {
     pub from: Pubkey,
     pub wallclock: u64,
-    pub slots: Vec<CompressedSlots>,
+    pub compressed_slots: Vec<u8>,
+    pub uncompressed_bytes: u64,
+    pub last_voted_slot: Slot,
     pub last_voted_hash: Hash,
     pub shred_version: u16,
 }
 
 impl Sanitize for RestartLastVotedForkSlots {
     fn sanitize(&self) -> std::result::Result<(), SanitizeError> {
-        if self.slots.is_empty() {
+        if self.last_voted_slot == 0 {
```

Comment:
why?

Reply:
Because you almost will never encode last_vote_slot to be 0?

Reply:
so is the case for slot 1 or 2 or 3. should we rule them out as well?
if nothing breaks with `last_voted_slot == 0` then I don't think sanitize should bother with that.

Reply:
Removed.